community_supplied/Chassis/chassis-env-info.yml

```diff
@@ -0,0 +1,10 @@
+---

```

Comment:
Please do not dump all the changes in a single Pull request. Request you to segregate them logically and submit appropriate requests

Reply:
Done